definitions.yaml

```diff
@@ -569,6 +569,19 @@ definitions:
         type: string
         description: The date and time when this account will expire
 
+  V4ModifyUserPassword:
+    type: object
+    required:
+      - new_password_base64
```

Comment:
Not sure about this required section.

`new_password_base64` is required for anyone accessing this endpoint, but `current_password_base64` only applies to users editing their own password and not admins.

Reply:
Why not make this same for both? Now admin will login via SSO anyway

Reply:
this is about the case where an admin wants to edit the password of someone else. 

so he doesn't know that persons 'current_password'

Reply:
I'd like to rename `V4ModifyUserPassword` to `V4ModifyUserPasswordRequest`.

We have only paid a bit of attention to this in the past. But as this can affect the APIs of generated clients, I'd like us to be consistent if we can.

Reply:
Thanks for noticing! Sorry I'm a bit inconsistent in noticing these details.

Reply:
That's what reviews are for :)